.github/workflows/tests.yml

```diff
@@ -34,27 +27,96 @@ jobs:
       with:
         context: .
         push: false
-        tags: appwrite-dev
+        tags: ${{ env.IMAGE }}
         load: true
         cache-from: type=gha
         cache-to: type=gha,mode=max
+        outputs: type=docker,dest=/tmp/${{ env.IMAGE }}.tar
         build-args: |
           DEBUG=false
           TESTING=true
           VERSION=dev
 
-    - name: Start Appwrite
-      run:  |
-        docker compose up -d
-        sleep 30
+    # - name: Pull Docker Images
+    #   run: docker compose pull --ignore-pull-failures --ignore-buildable
+    
+    # - name: List Files and images
+    #   run: |
+    #     docker compose config --images | sort -u
+    #     docker save $(docker compose config --images | sort -u) -o /tmp/allinone.tar
+
+    - name: Cache Docker Image
+      uses: actions/cache@v3
+      with:
+        key: ${{ env.IMAGE }}-${{ github.sha }}
+        path: /tmp/${{ env.IMAGE }}.tar
+
+    # - name: Upload Docker Image
+    #   uses: actions/upload-artifact@v3
+    #   with:
+    #     name: ${{ env.IMAGE }}
+    #     path: /tmp/${{ env.IMAGE }}.tar
```

Comment:
Leftover, above is same thing but with cache instead of uploading

Reply:
removed